src/components/HdIcon.vue

```diff
@@ -53,6 +67,27 @@ export default {
 
       return svg;
     },
+    addAccessibilityTags(svg) {
+      const randId = Math.floor(Math.random() * 1000);
+      const attrs = [];
+      if (this.title) {
+        const titleId = `${this.id || randId}-title`;
+        const titleTag = document.createElementNS('http://www.w3.org/2000/svg', 'title');
+        titleTag.innerHTML = this.title;
+        titleTag.id = titleId;
+        svg.insertBefore(titleTag, svg.firstChild);
+        attrs.push(titleId);
+      }
+      if (this.description) {
+        const descId = `${this.id || randId}-desc`;
+        const descTag = document.createElementNS('http://www.w3.org/2000/svg', 'desc');
+        descTag.id = descId;
```

Comment:
```suggestion
        descTag.id = descId;
        descTag.textContent = this.description;
```
I think you forgot to actually add the content to the element.